website/docs/concepts/teaming.md

```diff
@@ -4,4 +4,36 @@ title: Collaborate with Teams
 sidebar_label: Teaming
 ---
 
-## Coming soon
+The litmus portal has a built in teaming feature to facilitate collaboration between users using project level role access, the core concepts of which are discussed below.
```

Comment:
Can we update litmus portal to chaos center? 
Cc: @arkajyotiMukherjee

Reply:
Yes, Litmus Portal was the internal name...we should update it to `ChaosCenter`